lib/tasks/test_directories.rake

```diff
@@ -1,20 +1,16 @@
 namespace :test do
-  desc "test lib"
-  Rake::TestTask.new(:lib) do |t|
-    t.libs << "test"
-    t.pattern = 'test/lib/**/*_test.rb'
-    t.verbose = true
-  end
+  task run: %w(
+    test:units test:functionals test:generators test:integration
+    test:services test:libs
+  )
```

Comment:
I made these changes already in https://github.com/hummingbird-me/hummingbird/commit/a800e1c48af79d49b95ed277fd0de67a47e7ca6f already on the oauth branch